src/libcore/num/wrapping.rs

```diff
@@ -7,7 +7,13 @@
 // <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
+
 #![allow(missing_docs)]
+#![allow(deprecated)]
+
+#[unstable(feature = "core", reason = "may be removed, renamed, or relocated")]
+#[deprecated(since = "1.0.0", reason = "moved to std::num")]
+pub use super::Wrapping;
```

Comment:
Due to deprecation warnings not working on reexports, I'd be fine just removing this for now.